docs/source/gbd2017_models/risks/smoking/index.rst

```diff
@@ -4,58 +4,170 @@
+
+Pack-years among current smokers should be assigned as a categorical exposure value that is assigned in the following way:
 
-    Note: prevalence of smoking is forcasted as well as pack-years for current smokers
+- Each simulant gets an individual propensity value, which is a random value between 0 and 1 (uniformly distributed). This propensity value does not change over the course of the simulant's life.
 
-    Years since quitting will be used for initialization only
+- This propensity will determine the exposure category such that the probability of occupying an exposure category will be defined as the value for that category divided by the sum of values across all exposure categories for each age/sex/year/location group.
 
-    Pack-year smoking history exposure will be assigned to former smokers by looking up the pack-year history among current smokers in the year that they quit (assumption/limitation of model; potential error here should be investigated)
+- The *point* value for pack-year exposure assigned to each simulant should be the **minimum** value of the exposure bin to which the simulant was assigned.
 
+.. note::
+
+  See the notebook `here <https://github.com/ihmeuw/vivarium_data_analysis/pull/95>`_ that demonstrates this method most closely replicates the smoking PAF (most likely because the higher exposure bins are very large and likely right skewed).
+
+- Pack-year exposure values are updated when the exposure distribution for that simulant's demographic group changes (each year of the simulation and/or when a simulant ages into a new age group).
+
+.. note::
+
+  This method has the possibility that some simulants will have *decreases* in their pack-year exposure value, which is a measure of cumulative cigarettes smoked and therefore should logically increase monotonically.
+
+Pack-year exposure data are stored here: `/ihme/csu/swiss_re/forecast/py_forecast_draws.csv` and are age-, sex-, location-, and year-specific. The units of this file are number of individuals in each exposure bin per person-year of the *general population.* 
+
+Pack-years Among Former Smokers
++++++++++++++++++++++++++++++++
+
+Pack-years among former smokers should be assigned in a similar way to pack-years among current smokers, although the exposure should be sampled from the pack-year distribution among current smokers *the last year that the former smoker was a current smoker*. In other words, the year equal to the current year minus the simulant's years since quitting (see section below).
+
+The pack-year exposure data for former smokers should not change for the entire duration that a simulant is classified as a former smoker. 
+
+If a former smoker simulant becomes a current smoker, that simulant's pack-year exposure should be updated to reflect the pack-year exposure distribution in the current year (the year in which the former smoker becomes a current smoker).
```

Comment:
So that means we allow for simulants quit smoking with low pack-years , but a high pack-years when they become current smoker.

Reply:
Yes, we are not enforcing consistent exposure histories for individual simulants, but rather enforcing consistency with population level exposure forecasts.